modules/tinymce/src/themes/silver/test/ts/module/CommonMailMergeFetch.ts

```diff
@@ -0,0 +1,107 @@
+import { TestHelpers } from '@ephox/alloy';
+import { Toolbar } from '@ephox/bridge';
+import { Arr } from '@ephox/katamari';
+
+export const fetchMailMergeData = (settings: { collapseSearchResults: boolean }, store: TestHelpers.TestStore): Toolbar.ToolbarMenuButtonSpec['fetch'] => (callback, fetchContext) => {
+  const makeMailMerge = (info: { value: string; title?: string}) => ({
+    type: 'menuitem',
+    text: info.title ?? info.value,
+    onAction: () => {
+      store.adder('Triggering: ' + info.value)();
+    }
+  });
+
+  const makeCategory = (title: string, items: any[]) => ({
```

Comment:
Should this be `any`? I would have expected this to have types from bridge.

Reply:
- [ ] use types in Testing utility for fetching